app/controllers/provider_foreman_controller.rb

```diff
@@ -319,25 +319,8 @@ def load_or_clear_adv_search
   end
 
   def x_show
-    @explorer = true
     tree_record unless unassigned_configuration_profile?(params[:id])
-
-    respond_to do |format|
-      format.js do
-        unless @record
-          check_for_unassigned_configuration_profile
```

Comment:
@martinpovolny https://github.com/ManageIQ/manageiq/pull/12313/files#diff-dd586a3167b910b76243edbecbbf0de5L328 is not part of new common method https://github.com/ManageIQ/manageiq/pull/12313/files#diff-197a2eb3c02f33f29c3cd0ecc34082c4R155